DEVELOPMENT.md

```diff
@@ -15,38 +15,215 @@
+         - [Automatically Recompile Code When it Changes](#automatically-recompile-code-when-it-changes)
+      + [Troubleshooting](#troubleshooting)
+         - [Yarn Package Issues](#yarn-package-issues)
+            * [If Yarn Keeps Updating to The Latest Version](#if-yarn-keeps-updating-to-the-latest-version)
+         - [Yarn Test Issues](#yarn-test-issues)
+            * [Data Editor Opens Test Case Failing](#data-editor-opens-test-case-failing)
+            * [TLS Certificate Issues](#tls-certificate-issues)
+         - [Debugging Issues](#debugging-issues)
+            * [SELinux Port Functionality](#selinux-port-functionality)
+            * [Windows Yarn Test Window Not Loading Extensions](#windows-yarn-test-window-not-loading-extensions)
+         - [Re-trying Verification After Errors](#re-trying-verification-after-errors)
+   * [Building the Documentation](#building-the-documentation)
+      + [Install Pandoc](#install-pandoc)
+      + [Command to build the Documentation](#command-to-build-the-documentation)
+   * [Reviewing and Verifying Dependency Bot Updates](#reviewing-and-verifying-dependency-bot-updates)
+   * [Testing Information](#testing-information)
+   * [Monitoring Project Status](#monitoring-project-status)
+   * [Thank you for your interest in contributing to this project!](#thank-you-for-your-interest-in-contributing-to-this-project)
+
+## Welcome
+
+This guide contains developer-oriented instructions on how to setup your local development environment for contributing to this project. 
+
+If you would like the latest stable release of the extension, please reference the [README file](README.md). 
+
+The project is currently growing, please refer to the [wiki](https://github.com/apache/daffodil-vscode/wiki) for an overview of what the extension includes. There is also user documentation on the right pane with release specific use case guides. 
+
+## Prerequisites 
+
+### Install Git
+If your system doesn’t have Git installed, [install Git](https://git-scm.com/downloads). Select the appropriate operating system and follow the listed instructions. 
+To verify git is installed on your system, enter git -v into your system’s terminal and it should display the version of git if properly installed.
+
+### Install Visual Studio Code (VSCode)
+
+Install VSCode per [official VSCode documentation](https://code.visualstudio.com/docs/setup/setup-overview):
+* [Windows](https://code.visualstudio.com/docs/setup/windows)
+* [Linux](https://code.visualstudio.com/docs/setup/linux)
+* [MacOS](https://code.visualstudio.com/docs/setup/mac)
+
+## Installing Build Requirements
+This section will provide describe what requirements are needed to build with a step-by-step guide to assist developers of varying skill levels. The steps provide recommended methods for installing build requirements, but don't have to be followed exactly one-to-one.   
+
```

Comment:
This sentence needs to be modified:
This section will provide describe what requirements are needed to build with a step-by-step guide to assist developers of varying skill levels.